### ISSUE TITLE
[Interop] Zero-knowledge circuits

### ISSUE THREAD

**Issue by biscuitdey** [eea-oasis/baseline#724]
## Overview

Circuits need to verify non-inclusion merkle proof and ecdsa signature.

**Comment by Therecanbeonlyone1969** [comment]
@fleischr Agreed upon Prover: Plonk (no Groth16), BRI-1 needs to allow for the upload of custom circuits for the Interop.
